packages/manager/modules/iam/src/components/identities/userIdentities/userIdentities.controller.js

```diff
@@ -29,6 +32,45 @@ export default class userIdentitiesController {
     }
   }
 
+  openSelectUsersModal() {
+    this.isSelectingUsers = true;
+  }
+
+  closeSelectUsersModal() {
+    this.isSelectingUsers = false;
+  }
+
+  async getUserList() {
```

Comment:
as you're using it in the html template, prefer using `.then()` syntax to avoid having issue with `$digest` lifecycle